src/network.c

```diff
@@ -940,8 +976,8 @@ int network_init(server *srv) {
 		SSL_CTX_set_mode(s->ssl_ctx, SSL_CTX_get_mode(s->ssl_ctx) | SSL_MODE_ACCEPT_MOVING_WRITE_BUFFER);
 
 # ifndef OPENSSL_NO_TLSEXT
-		if (!SSL_CTX_set_tlsext_servername_callback(s->ssl_ctx, network_ssl_servername_callback) ||
-		    !SSL_CTX_set_tlsext_servername_arg(s->ssl_ctx, srv)) {
+		if (SSL_CTX_set_tlsext_servername_callback(s->ssl_ctx, network_ssl_servername_callback) != 1 ||
+		    SSL_CTX_set_tlsext_servername_arg(s->ssl_ctx, srv) != 1) {
 			log_error_write(srv, __FILE__, __LINE__, "ss", "SSL:",
```

Comment:
Unnecessary change.  These macros are simple accessors that set a value and return 1.